src/scripts/item-review/item-review.html

```diff
@@ -32,11 +32,11 @@
   <div ng-if="$ctrl.procon" class="community-review--procon">
     <div>
       <div translate="DtrReview.Pros"></div>
-      <div ng-repeat="pro in $ctrl.aggregate.pros" ng-bind='pro'></div>
+      <div ng-repeat="pro in $ctrl.aggregate.pros" ng-bind="pro"></div>
     </div>
     <div>
       <div translate="DtrReview.Cons"></div>
-      <div ng-repeat="pro in $ctrl.aggregate.cons" ng-bind='pro'></div>
+      <div ng-repeat="pro in $ctrl.aggregate.cons" ng-bind="pro"></div>
```

Comment:
@48klocs should this be `<div ng-repeat="con in $ctrl.aggregate.cons" ng-bind="con"></div>`